mava/utils/jax_training_utils.py

```diff
@@ -37,13 +40,50 @@ def init_norm_params(stats_shape: Tuple) -> Dict[str, Union[jnp.array, float]]:
     return stats
 
 
+def construct_norm_axes_list(
+    start_axes: int, axes_list: List[Any], obs_shape: Tuple
+) -> Tuple[slice, ...]:
+    """Construt a list of Tuples containing the features on which to apply normalisation
+
+    start_axes (int) --- default axes from which to start
+    axes_list (List[int]) --- user specified
+    obs_shape (Tuple) --- observations shape
+
+    Returns:
+        axes_list: a tuple to be used with np.r_
+    """
+
+    if len(axes_list) == 0:
+        return_list = [slice(start_axes, obs_shape[0])]
+    else:
+        return_list = []
+        indices = []
+        for x in axes_list:
+            if type(x) == tuple or type(x) == list:
+                element = slice(x[0] + start_axes, x[1] + start_axes)
+                indices.append(x[1] + start_axes)
+            else:
+                element = slice(x + start_axes, x + start_axes + 1)
+                indices.append(x + start_axes + 1)
+            return_list.append(element)
+
+        assert (
```

Comment:
Minor detail, but can we please raise an error here instead of the assert?

Reply:
Modified to raise a ValueError